Kernel/CMakeLists.txt

```diff
@@ -464,7 +464,6 @@ elseif("${SERENITY_ARCH}" STREQUAL "aarch64")
         Arch/aarch64/init.cpp
         Arch/aarch64/InterruptManagement.cpp
         Arch/aarch64/Interrupts.cpp
-        Arch/aarch64/kprintf.cpp
```

Comment:
Should we rm this file now?

Reply:
…oh I see, it's in the next commit. Squash it in this one.

Reply:
Yup, done!